Source/DafnyDriver/Commands/DafnyCliTests.cs

```diff
@@ -42,8 +42,11 @@ public static class DafnyCliTests {
     // We do not want output such as "Compiled program written to Foo.cs"
     // from the compilers, since that changes with the target language
     "/compileVerbose:0",
-      
-    // Set a default time limit, to catch cases where verification time runs off the rails
+
+    // Set a default resource limit, to catch cases where verification runs off the rails
+    "/rlimit:50000000",
```

Comment:
Shouldn't this be `50000` instead, since this one does the X1000?

Reply:
Good catch.

Reply:
Oh, actually, no. Boogie no longer does the multiplication, and it handles this flag directly. It's one of many reasons to deprecate it. I think I'll implement the deprecation warning as part of fixing #4204.

Reply:
Oh I didn't realize that was true for the CLI flag. Should we update https://github.com/dafny-lang/dafny/blob/master/docs/dev/news/4975.fix to call both out?

Reply:
Yep, I'll do that in a separate PR.